Tools/TestWebKitAPI/Tests/WebKitGtk/TestPrinting.cpp

```diff
@@ -416,7 +416,7 @@ class PrintCustomWidgetTest: public WebViewTest {
     void jumpToCustomWidget()
     {
         // Jump back to the GtkNotebook
-        keyStroke(GDK_KEY_Tab, GDK_SHIFT_MASK);
+        keyStroke(GDK_KEY_Tab, { WebViewTest::Modifiers::Shoft });
```

Comment:
I suppose this would be `Shift`:

```suggestion
        keyStroke(GDK_KEY_Tab, { WebViewTest::Modifiers::Shift });
```